projects/addon-doc/components/api/api-item.component.ts

```diff
@@ -71,7 +71,7 @@ export class TuiDocAPIItem<T> implements OnInit {
     @Output()
     public readonly valueChange = new EventEmitter<T>();
 
-    public ngOnInit(): void {
+    public ngAfterViewInit(): void {
```

Comment:
Or should it be constructor? Since it's a snapshot that doesn't change

Reply:
@waterplea 

Unfortunately, this function also uses input properties – `constructor` is not an option

https://github.com/taiga-family/taiga-ui/blob/499ac55f0b30a3d5f1f6cdeeeafdd4bd64ec13b2/projects/addon-doc/components/api/api-item.component.ts#L59-L60

https://github.com/taiga-family/taiga-ui/blob/499ac55f0b30a3d5f1f6cdeeeafdd4bd64ec13b2/projects/addon-doc/components/api/api-item.component.ts#L94-L95